manifests/node/kubelet.pp

```diff
@@ -6,6 +6,7 @@
 # @param ca_cert path to the ca cert
 # @param cert path to node cert file
 # @param cert_path path to cert files
+# @param labels lables to be setted on the node
```

Comment:
`# @params labels node labels to be set on initial registration`

Or something similar, just to make it clearer that this is something which Kubelet only uses for bootstrap/initial registration